app/src/main/java/io/apicurio/registry/metrics/RestMetricsResponseFilteredNameBinding.java

```diff
@@ -0,0 +1,14 @@
+package io.apicurio.registry.metrics;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.ws.rs.NameBinding;
```

Comment:
Same here, I would replace this name by something more related to its behaviour.